cmd/cstor-pool-mgmt/pool/pool.go

```diff
@@ -326,6 +328,24 @@ func CheckForZreplContinuous(ZreplRetryInterval time.Duration) {
 			if PoolAddEventHandled && strings.Contains(string(out), StatusNoPoolsAvailable) {
 				break
 			}
+
+			// Command   : $ sudo zpool get capacity -Hp
+			// raw output: vol1	  capacity	  12%	-
+
+			rawOut, err := RunnerVar.RunCombinedOutput(PoolOperator, "get", "capacity", "-Hp")
+			if err != nil {
+				glog.Errorf("zpool get capacity returned error in zrepl capacity check, error: %v", err)
+			}
+			if len(rawOut) != 0 {
+				// split the rawOut by skipping spaces and get slice data
+				data := strings.Fields(string(rawOut))
+				usedCapacity := strings.TrimSuffix(data[2], "%")
```

Comment:
'%' will not come if -p is used

Reply:
it's coming in my setup, attached image for reference
![image](https://user-images.githubusercontent.com/19219723/52031160-2972c480-2541-11e9-9e5c-e18cd9e4b52b.png)